src/middlewares/__tests__/createInsightsMiddleware.ts

```diff
@@ -103,16 +103,16 @@ describe('insights', () => {
       });
     });
 
-    it('does not throw when an event is sent right after the creation in UMD', () => {
+    it('does not throw without userToken in UMD (1)', () => {
```

Comment:
What are these `(1)` and `(2)` suffixes for? Can we name the tests based on the scenario?

Reply:
yup https://github.com/algolia/instantsearch.js/pull/4724/commits/3b7f5674611358ab0b75391036adae959e83a03d